xbmc/settings/Settings.cpp

```diff
@@ -666,6 +666,7 @@ void CSettings::InitializeISettingCallbacks()
   settingSet.clear();
   settingSet.insert("karaoke.export");
   settingSet.insert("karaoke.importcsv");
+  settingSet.insert("musiclibrary.hideallitems");
```

Comment:
You don't need these as you don't do anything explicitly (i.e. you don't handle them in one of the `ISettingCallback` callbacks) if the settings are changed.